modules/sdk/project-templates/project-templates-activator/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+def projectDir = Paths.get(request.getOutputDirectory(), request.getArtifactId())
+
+def buildGradleFile = projectDir.resolve('build.gradle')
+
+if (Files.exists(buildGradleFile)) {
+	Files.delete(buildGradleFile)
```

Comment:
Isn't it easier to just use [`Files.deleteIfExists`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#deleteIfExists(java.nio.file.Path))?